src/main/resources/logback-spring.xml

```diff
@@ -15,6 +15,21 @@
         </encoder>
     </appender>
 
+    <appender name="EmergencyAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="org.candlepin.insights.logging.CloudWatchEmergencyFilter" />
+        <encoder>
+            <pattern>%d{ISO8601} [thread=%thread] [%X{requestType}=%X{requestUuid}, org=%X{org}, csid=%X{csid}] %-5p %c - %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="CloudWatchAppender" class="com.j256.cloudwatchlogbackappender.CloudWatchAppender">
+        <region>${AWS_REGION}</region>
+        <logGroup>platform</logGroup>
+        <logStream>namespace</logStream>
+        <layout class="net.logstash.logback.layout.LogstashLayout" />
+        <appender-ref ref="EmergencyAppender" />
```

Comment:
I don't know that this is valid.  I've looked at the [docs](https://logback.qos.ch/manual/appenders.html) and it doesn't mention any place where one appender can reference the other.  I thought `appender-ref` was for use under the `root` element.  Assuming this is intentional, what's the behavior here?  The CloudWatchAppender inherits the filter and encoder from EmergencyAppender?

Reply:
@awood I added some comments. Will happily port these over to the rhsm-subscriptions repo too once we're happy with this PR.